docs/src/rules/no-irregular-whitespace.md

```diff
@@ -39,27 +39,27 @@ This rule disallows the following characters except where the options allow:
 \u000B - Line Tabulation (\v) - <VT>
 \u000C - Form Feed (\f) - <FF>
 \u00A0 - No-Break Space - <NBSP>
-\u0085 - Next Line
-\u1680 - Ogham Space Mark
+\u0085 - Next Line - <NEL>
+\u1680 - Ogham Space Mark - <OGSP>
 \u180E - Mongolian Vowel Separator - <MVS>
```

Comment:
```suggestion
\u180E - Mongolian Vowel Separator - <MVS> - <MVSEP>
```

Reply:
@amareshsm is MVSEP a common abbreviation for this character? I couldn't find it in any documents.

Reply:
In documentation dealing with Mongolian script, a more explicit term like <MVSEP> could provide better clarity over the standard <MVS> but is not considered a common abbreviation. let's ignore this comment.